ui/app/serializers/task-group.js

```diff
@@ -8,6 +8,11 @@ export default class TaskGroup extends ApplicationSerializer {
   mapToArray = ['Volumes'];
 
   normalize(typeHash, hash) {
+    if (hash.Services) {
+      hash.Services.forEach((service) => {
+        service.GroupName = hash.Name;
+      });
+    }
```

Comment:
not needed in the task serializer, as services there already get populated with `TaskName` props.